lib/mongoid/contextual/mongo.rb

```diff
@@ -168,14 +168,12 @@ def each(&block)
       #   b-tree indexes.
       #
       # @param [ Hash | Object | false ] id_or_conditions an _id to
-      #   search for, a hash of conditions, or false.
+      #   search for or a hash of conditions.
       #
       # @return [ true | false ] If the count is more than zero.
-      #   Always false if passed false.
       def exists?(id_or_conditions = nil)
         case id_or_conditions
         when nil then !!(view.projection(_id: 1).limit(1).first)
```

Comment:
@Neilshweky did you see my comment re: the reasons for Rails behavior? I really think we should change this to use :none rather than nil as default value, and make `nil` return `false`.

Reply:
hmmm point taken... I think it might be smart to take Rails' lead on this one... 

use :none for default and then automatically return false on "falsey" value